### PR TITLE
Using azure-cli python sdk

### PR DESCRIPTION
**Pull request by ds-ms**

### CODE REVIEW

File: Tasks/AzureCLI/azureclitask.ts [hunk 1]
```diff
@@ -91,131 +91,52 @@ export class azureclitask {
...
-            }
-            else {
-                connectedService = tl.getInput("connectedServiceName", true);
-                this.logoutAzureClassic(connectedService);
-            }
+            tl.execSync("az", " account clear");
         }
-        catch(err){
+        catch (err) {
             // task should not fail if logout doesn`t occur
```
Comment (sachinma) [inline Tasks/AzureCLI/azureclitask.ts]: You should probably add a message (warning probably)

File: Tasks/AzureCLI/task.json [hunk 2]
```diff
@@ -16,11 +16,11 @@
     ],
     "demands": [],
     "version": {
-        "Major": 0,
-        "Minor": 2,
-        "Patch": 8
+        "Major": 1,
+        "Minor": 0,
+        "Patch": 0
```
Comment (sachinma) [inline Tasks/AzureCLI/task.json]: Do we need to add the message describing changes in 1.*

File: Tests-Legacy/L0/AzureCLI/azureclitaskPass.json [hunk 3]
```diff
@@ -26,34 +26,26 @@
       "code": 0,
       "stdout":"bash executed successfully"
     },
-    "/usr/local/bin/azure config mode asm": {
+    "/usr/local/bin/az account import subscriptions.publishsettings": {
```
Comment (sachinma) [inline Tests-Legacy/L0/AzureCLI/azureclitaskPass.json]: Why do we have a test for publishsetting?

File: Tasks/AzureCLI/azureclitask.ts (hunk 1 above)
Comment (ds-ms) [inline Tasks/AzureCLI/azureclitask.ts]: Will add it.

File: Tasks/AzureCLI/task.json (hunk 2 above)
Comment (ds-ms) [inline Tasks/AzureCLI/task.json]: @sachinma waiting for Usha on that.

File: Tests-Legacy/L0/AzureCLI/azureclitaskPass.json (hunk 3 above)
Comment (ds-ms) [inline Tests-Legacy/L0/AzureCLI/azureclitaskPass.json]: These aren't used. will remove them

**Pull request merged**